doc/sphinx/.sphinx/pa11y.json

```diff
@@ -1,9 +1,11 @@
 {
-  "chromeLaunchConfig": {
-    "args": [
-      "--no-sandbox"
-    ]
+  "defaults": {
+    "chromeLaunchConfig": {
+      "args": [
+	"--no-sandbox"
+      ]
+    }
```

Comment:
This something that should be contributed [upstream](https://github.com/canonical/sphinx-docs-starter-pack/blob/main/.sphinx/pa11y.json)?

Reply:
This is specifically for `pa11y-ci`, so it depends on whether or not we decide to upstream `pa11y-ci` (see discussion below)